ynput/ayon-third-party
# Added descriptions for libraries settings.

Added descriptions for addon's setting for better clarity and fixed small issue in the info text for OIIO category.

Resolves [https://github.com/ynput/ayon-third-party/issues/11](url)

Testing notes:

1) Build addon package and upload to the server
2) Check addon's settings categories and added descriptions if correct.
